src/hooks/useTokenOperations.ts

```diff
@@ -0,0 +1,33 @@
+import { PricedBalance } from "config/types";
+import useAppTranslation from "hooks/useAppTranslation";
+import { useClipboard } from "hooks/useClipboard";
+
+/**
+ * Hook that provides token-related operations
+ */
+export const useTokenOperations = () => {
```

Comment:
Considering the stellar network context I think naming it as `tokenOperations` sounds a bit misleading as the first thing that comes to mind is operations from stellar transactions using a token, we should probably use another name here

Reply:
maybe `useAssetActions` ?

Reply:
I think I'd also rename `copyTokenAddress` to `copyAssetAddress` as on the UI we're always referring it as "Asset". @Brunonascdev what do you think?